sources/hachiko/src/main.rs

```diff
@@ -56,11 +56,22 @@ async fn get_config() -> Result<Config> {
 async fn main() {
     SimpleLogger::init(LevelFilter::Info, LogConfig::default())
     .expect("unable to configure logger");
-    let should_wait_value = get_config().await.unwrap().should_wait;
+    let config_parse = get_config().await.unwrap();
```

Comment:
`unwrap` leaves us with a less meaningful output than explicit handling, so it would be better to use `context`  which is intended to be a human-readable, user-facing explanation for the error that has occurred.